workflow/rocoto/gfs_tasks.py

```diff
@@ -944,6 +944,23 @@ def arch(self):
 
         return task
 
+    # Cleanup
+    def cleanup(self):
+        deps = []
+        if 'enkf' in self.cdump:
+            dep_dict = {'type': 'metatask', 'name': 'enkfgdaseamn'}
```

Comment:
Do we need to add a dependency on `enkfgfseamn` when the early cycle EnKF is also ON?